terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1732,6 +1766,35 @@ def qq(*words):
                     'Owner': config.owner
                 }
             }
+        },
+        'aws_cloudwatch_metric_alarm': {
+            **{
+                'gitlab_' + resource: {
+                    'alarm_name': 'azul-gitlab_' + resource,
+                    'comparison_operator': 'GreaterThanOrEqualToThreshold',
+                    'evaluation_periods': period,
+                    'period': 60 * 10,
+                    'metric_name': metric,
+                    'namespace': 'CWAgent',
+                    'dimensions': dimensions | {
+                        'InstanceId': '${aws_instance.gitlab.id}',
+                    },
+                    'statistic': stat,
+                    'threshold': threshold,
+                    'treat_missing_data': 'missing',
+                    **{
+                        state + '_actions': ['${data.aws_sns_topic.monitoring.arn}']
+                        for state in ('insufficient_data', 'alarm', 'ok')
+                    },
+                } for resource, period, metric, dimensions, stat, threshold in
+                [
+                    # FIXME: Add `mem_used_percent` alarm
+                    #        https://github.com/DataBiosphere/azul/issues/5139
+                    ('data_disk_use', 1, 'disk_used_percent', {'path': '/mnt/gitlab', 'fstype': 'ext4'}, 'Maximum', 75),
```

Comment:
Unfortunately, without `'fstype'` in the dimensions, the alarm won't configure properly, so it must be included.